apis/vshn/v1/dbaas_vshn_keycloak.go

```diff
@@ -84,11 +84,11 @@ type VSHNKeycloakServiceSpec struct {
 	// +kubebuilder:default="/"
 	RelativePath string `json:"relativePath,omitempty"`
 
-	// +kubebuilder:validation:Enum="23"
-	// +kubebuilder:default="23"
+	// +kubebuilder:validation:Enum="23.0.5-202402021353-44-af6cea11"
```

Comment:
I've asked them, if they can multitag the images, this looks like a great source for user errors.

Reply:
Inventage currently only provides this image to us. So yes we only have this one at the moment. It can be extended in the future. We're in contact with Inventage.

This is currently still very early stage and many changes will come soon.

Reply:
ok, fine for me